src/project/qml/MuseScore/Project/internal/NewScore/TitleListView.qml

```diff
@@ -86,9 +86,23 @@ Item {
         width: parent.width
     }
 
+    StyledTextLabel {
+        id: noResultsFoundHint
+
+        anchors.centerIn: parent
+
+        font: ui.theme.bodyBoldFont
+
+        text: "No results found."
```

Comment:
This should be

`qsTrc("project/newscore", "No results found")`

(to make this string translatable)